app/controllers/admin_events_controller.rb

```diff
@@ -74,4 +75,8 @@ def inform_applicants_field_of_interest
         UserNotificationsMailer.new_field_specific_event(@event, user).deliver_later
       end
     end
+
+    def tweet_event
+      Tweet.new(event_id: @event.id, published: false) if params[:approve][:tweet] == "0"
```

Comment:
This only adds new tweet to table if tweet checkbox is not checked as far as I can tell. Should there not be an else for the default published: true?